slack_bolt/app/app.py

```diff
@@ -335,6 +336,34 @@ def middleware(self, *args) -> None:
                     CustomMiddleware(app_name=self.name, func=middleware_or_callable)
                 )
 
+    # -------------------------
+    # Workflows: Steps from Apps
+
+    def step(
+        self,
+        callback_id: Union[str, Pattern, WorkflowStep],
+        save_callback_id: Optional[Union[str, Pattern]] = None,
+        edit: Optional[Union[Callable[..., Optional[BoltResponse]], Listener]] = None,
+        save: Optional[Union[Callable[..., Optional[BoltResponse]], Listener]] = None,
+        execute: Optional[
+            Union[Callable[..., Optional[BoltResponse]], Listener]
+        ] = None,
+    ):
+        """Registers a new Workflow Steps from Apps listeners."""
```

Comment:
Sorry, please change this docstring. This is grammatically incorrect and doesn't make sense.

Reply:
Changing to `"""Registers a new Workflow Step listener"""`